tls/ttls.c

```diff
@@ -1702,6 +1702,11 @@ ttls_write_change_cipher_spec(TlsCtx *tls, struct sg_table *sgt,
 	}
 }
 
+/**
+ * Do not allow to change the cipher spec actually to prevent CCS injection
```

Comment:
Is it a TODO note, or is it a description of the function that follows?
If it's a TODO, "TODO" string is missing.
If it's a description, then maybe it's better be where `ttls_parse_change_cipher_spec()` is called from. An FSM there calls `ttls_parse_change_cipher_spec()` once per handshake message, so that's where protection against CCS injection is.

Reply:
I updated the comment to make it clear. The thing is that the function doesn't changes the cipher specification as it is supposed to do, instead it just validates the input data. This is inherited from mbed TLS, but this comment just explains why the code is written in this way.

Reply:
That makes sense.